collect_app/src/main/java/org/odk/collect/android/utilities/SnackbarUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+
+Copyright 2018 Shobhit
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package org.odk.collect.android.utilities;
+
+import android.support.design.widget.Snackbar;
+import android.view.View;
+import android.widget.TextView;
+
+/**
+ * @author Shobhit Agarwal
+ */
+
+public final class SnackbarUtils {
+    private static final int LONG_DURATION_MS = 3500;
+
+    private SnackbarUtils() {
+
+    }
+
+    public static void showSnackbar(View view, String result) {
+        Snackbar snackbar = Snackbar.make(view, result.trim(), LONG_DURATION_MS);
```

Comment:
Maybe there’s a better name than `result`. `text`, or `message`?

Reply:
I’m assuming a general-purpose “show snackbar” method might show informational messages, and not just “results”.